### PR TITLE
Updating SIP Gateway documentation

### PR DESCRIPTION
**Pull request by ayushman2609**
Updating a few outdated details in the documentation

### CODE REVIEW

**Comment by ayushman2609** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: Teams/sip-gateway-configure.md [hunk 1]
```diff
@@ -195,15 +189,11 @@ To streamline your tasks, you can enroll SIP devices in the Teams admin center e
...
 
-7. Go to [https://microsoft.com/devicelogin](https://microsoft.com/devicelogin), and under **Enter code**, enter the SIP device's pairing code, and then select **Next**.
+7. Navigate to the authentication URL on the user's desktop or mobile browser and use corporate credentials to log in.
 
-8. On the Microsoft **Sign in** page, in the **Email or phone** field, enter the email address for the SIP device, and then select **Next**.
-
-9. On the **Password** page, enter the password for the email address for the SIP device, and then select **Sign in**.
-
-10. On the **Are you trying to sign in to Teams SIP devices gateway** page, select **Continue**.
+8. Enter the pairing code displayed in the **Sign in a user dialog** into the web authentication app to pair the SIP phone with the user's account. On a successful sign-in, which might take a while, the SIP phone will display the phone number and username, if the device supports it.
```
Comment (JohanFreelancer9) [inline Teams/sip-gateway-configure.md:196]:
```suggestion
8. Enter the pairing code displayed in the **Sign in a user** dialog into the web authentication app to pair the SIP phone with the user's account. On a successful sign-in, which might take a while, the SIP phone will display the phone number and username, if the device supports it.
```

File: Teams/sip-gateway-configure.md [hunk 2]
```diff
@@ -195,15 +189,11 @@ To streamline your tasks, you can enroll SIP devices in the Teams admin center e
 
 5.  On the **Provision devices** pane, under **Waiting for sign in**, select **Signed out**.
 
-6. In the **Sign in a user** dialog, copy or note the SIP device's pairing code.
+6. In the **Sign in a user** dialog, authentication URL and pairing code will be displayed.
```
Comment (JohanFreelancer9) [inline Teams/sip-gateway-configure.md:192]:
```suggestion
6. In the **Sign in a user** dialog, the authentication URL and pairing code will be displayed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ayushman2609, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by dariomws** [comment]
Hi @ayushman2609 @dstrome,

I have created the following PR based on feedback:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/9635/files

People are asking for more information about port and protocol:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9577
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9816

What do you think?

Thanks!

**Pull request merged**